DrrImageComputation/Widgets/qSlicerDrrImageComputationPlastimatchParametersWidget.cxx

```diff
@@ -289,8 +299,10 @@ void qSlicerDrrImageComputationPlastimatchParametersWidget::onUseExponentialMapp
     qCritical() << Q_FUNC_INFO << ": Invalid parameter node";
     return;
   }
-
+  d->ParameterNode->DisableModifiedEventOn();
   d->ParameterNode->SetExponentialMappingFlag(value);
+  d->ParameterNode->DisableModifiedEventOn();
```

Comment:
This should be DisableModifiedEventOff. Why is it necessary invoke DisableModifiedEventOn/DisableModifiedEventOff for all of these functions?

Reply:
Old version. Obsolete code. I have removed all unnecessary node update blocks, except one which is not part of Plastimatch DRR program.